platform-operator/helm-pod/main.go

```diff
@@ -380,6 +383,107 @@ func deleteHelmRelease(helmreleaseNS, helmrelease string) bool {
 	return ok
 }
 
+func updateCRDInstances(request *restful.Request, response *restful.Response) {
+	fmt.Printf("-- Inside updateCRDInstances...\n")
+
+	kind := request.QueryParameter("kind")
+	group := request.QueryParameter("group")
+	version := request.QueryParameter("version")
+	plural := request.QueryParameter("plural")
+	namespace := request.QueryParameter("namespace")
+	chartURL := request.QueryParameter("chartURL")
+	chartName := request.QueryParameter("chartName")
+	crName := request.QueryParameter("instance")
+
+	fmt.Printf("Kind:%s\n", kind)
+	fmt.Printf("Group:%s\n", group)
+	fmt.Printf("Version:%s\n", version)
+	fmt.Printf("Plural:%s\n", plural)
+	fmt.Printf("Namespace:%s\n", namespace)
+	fmt.Printf("ChartURL:%s\n", chartURL)
+	fmt.Printf("ChartName:%s\n", chartName)
+	fmt.Printf("CRName:%s\n", crName)
+
+	if _, errF := os.Stat("/" + chartName); os.IsNotExist(errF) {
+		fmt.Printf("Error: chart '%s' not installed", chartName)
+	}
+	
+	cmdRunnerPod := getKubePlusPod()
+	downloadUntarChartandGetName(chartURL, cmdRunnerPod, namespace)
+
+	apiVersion := group + "/" + version
+	fmt.Printf("APIVersion:%s\n", apiVersion)
+
+	ownerRes := schema.GroupVersionResource{Group: group, Version: version, Resource: plural}
+	fmt.Printf("GVR:%v\n", ownerRes)
+	
+	config, err := rest.InClusterConfig()
+	if err != nil {
+		panic(err.Error())
+	}
+	dynamicClient, _ := dynamic.NewForConfig(config)
+
+	crdObjList, err := dynamicClient.Resource(ownerRes).Namespace("").List(context.Background(), metav1.ListOptions{})
```

Comment:
Instead of empty Namespace (""), shouldn't this be using the "namespace" parameter?

Reply:
Yes, that would be better. Fixed.